monitoring/statsd/metrics.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from datadog import DogStatsd
+
+
+statsd_host = os.environ.get("STATSD_ENDPOINT")
+statsd_port = int(os.environ.get("STATSD_PORT"))
+statsd = DogStatsd(host=statsd_host, port=statsd_port)
+
+REQUEST_LATENCY_METRIC_NAME = "django_request_latency_seconds"
+REQUEST_COUNT_METRIC_NAME = "django_request_count"
+NUM_SUBMISSIONS_IN_QUEUE = "num_submissions_in_queue"
+NUM_PROCESSED_SUBMISSIONS = "num_processed_submissions"
+
+
+def set_statsd_gauge(metric_name, tags, set_value):
+    statsd.gauge(metric_name, set_value, tags=tags)
```

Comment:
Why is there a gauge and increment method? We only need to increment the metric right?